packages/semantic-ui/package.json

```diff
@@ -31,7 +31,7 @@
     "nanoid": "^3.1.23",
     "react": ">=16",
     "semantic-ui-css": "^2.4.1",
-    "semantic-ui-react": "^0.87.3"
+    "semantic-ui-react": "1.3.1"
```

Comment:
Are there any breaking changes in this version bump that would make it different to use @rjsf/semantic-ui? If so, we might need to bump the rjsf version to a major version as well.

Reply:
Thanks for making the issue @jacqueswho! I think this is the only remaining outstanding issue before we can get this merged.

Reply:
@epicfaace according to [this ](https://github.com/Semantic-Org/Semantic-UI-React/releases) no.

Reply:
@epicfaace I have updated the getSemanticProps function, to make it more simpler